net-print/kyocera-1x2x-mfp-driver/kyocera-1x2x-mfp-driver-1.1203-r1.ebuild

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Printer descriptions (PPDs) and filters for Kyocera 1x2x MFP"
+HOMEPAGE="http://www.kyoceramita.it/"
+SRC_URI="LinuxDrv_${PV}_FS-1x2xMFP.zip"
+
+LICENSE="kyocera-mita-ppds"
+SLOT="0"
+
+# TODO: Someone please test the ebuild on ~x86.
+KEYWORDS="-* ~amd64 ~x86"
+
+IUSE_LINGUAS=(ar cs de el en es fr he hu it ko pl pt ro ru th tr vi zh_CN zh_TW)
```

Comment:
`LINGUAS` is deprecated because it violates EAPI 5+ principles. Use `L10N` instead.